packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties

```
@@ -1,7 +1,8 @@
 fielderror.screenreader.text = Error
 fieldlabel.optional.text = Optional
 fieldlabel.required.text = Required
-passwordfield.icon.arialabel = toggle password visibility
+passwordfield.icon.arialabel.show = Show password
+passwordfield.icon.arialabel.hide = Hide hassword
```

I'd rather call these `label` and not `arialabel` because we treat them both the same from an accessibility standpoint.

```suggestion
passwordfield.icon.label.show = Show password
passwordfield.icon.label.hide = Hide password
```